packages/mobile/src/home/NotificationBox.tsx

```diff
@@ -145,6 +149,29 @@ function useSimpleActions() {
     })
   }
 
+  if (inviteRewardsEnabled) {
```

Comment:
Just to confirm, this card is not dismissable?

Reply:
lol yeah 😅 i think this experiment is going to be super short (a few days) so hopefully not a big issue. i'll double check though!